lib/ScrollControls/ScrollControls.js

```diff
@@ -105,6 +103,9 @@ var ScrollControls = module.exports = kind(
 		}
 		if (this.scroller) {
 			this.scroller.on('scrollabilityChanged', this._updateScrollability);
+			this.subControls.forEach(this.bindSafely(function (nom) {
```

Comment:
No, you can flatten this out (remove the array and just explicitly set the scroller property on each subcontrol). I don't remember why I did this, but my guess is that at some earlier point I may have been performing another operation on each of the subcontrols and didn't like the brittleness of having to update multiple places in the code if anything changed.